job/common/src/main/java/alluxio/job/util/TimeSeries.java

```diff
@@ -80,8 +80,7 @@ public void record(long timeNano) {
    */
   public void record(long timeNano, int numEvents) {
     long leftEndPoint = bucket(timeNano);
-    mSeries.put(leftEndPoint,
-        (mSeries.containsKey(leftEndPoint) ? mSeries.get(leftEndPoint) : 0) + numEvents);
+    mSeries.put(leftEndPoint, (mSeries.getOrDefault(leftEndPoint, 0)) + numEvents);
```

Comment:
You can remove the parentheses around `mSeries.getOrDefault(leftEndPoint, 0)` - they are no longer needed

```java
mSeries.put(leftEndPoint, mSeries.getOrDefault(leftEndPoint, 0) + numEvents);
```

Reply:
Thanks for your advice.